arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-morocco.dts

```diff
@@ -1008,6 +1008,31 @@ spd_ ## bus ## _ ## index: spd@addr,4cc5118 ## index ## 000 { \
 
 &gpio0 {
 	status = "okay";
+
+	/*
+	 * by default the direction of the SEL_P0_SEC_I2C_ROT_BMC (GPIO B0)
+         * is "in". It should be set to "out" and value of "1" for the
+         * I2C bus to be owned by the BMC.
+         */
+	p0_sec_i2c {
```

Comment:
This is only the mux selection change between PROT and BMC. Changes to use I2C8 and I2C9 instead of I3C0 and I3C1 will be forthcoming next?